### PR TITLE
Add Login page

### PR DESCRIPTION
**Pull request by katsiarynalashcheuskaya**
## Summary:

Create a UI for the Login page.

## Requirements:

- Follow the design layout of the project in Figma for creating responsive UI using MUI library.
- Create a stub for the submit button.

## UI/UX Outcome:

> 
Screenshot 2023-10-30 at 11 10 11 PM

### CODE REVIEW

File: src/components/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                        </Typography>
+                        <Box component="form" noValidate onSubmit={handleSubmit} sx={{mt: 1}}>
+                            <TextField
+                                margin="normal"
+                                required
+                                fullWidth
+                                id="email"
+                                label="Email Address"
+                                name="email"
+                                autoComplete="email"
```
Comment (ezbc) [inline src/components/LoginPage.jsx:68]: I'm unfamiliar with this autocomplete feature! Could you explain what it does?

**Comment by ezbc** [comment]
I appreciate the description of the PR! I left a couple small comments and once the login page is rendered on the app I'll be able to complete the review :+1:

File: src/components/LoginPage.jsx (hunk 1 above)
Comment (katsiarynalashcheuskaya) [reply src/components/LoginPage.jsx:68]: Autocomplete is an HTML attribute available for input elements. When used with the input tag (in our case, the MUI TextField component), it allows the browser to auto-fill the input with values it has received from previous attempts to fill in this input. Just like when logging into any website you've used before, you can easily select your email or other data once you start filling in this input.

Comment (ezbc) [reply src/components/LoginPage.jsx:68]: Thank you!

**Pull request merged**